src/components/EmailNotifications/StatusInfo.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { Button, GU, textStyle, useTheme } from '@aragon/ui'
+import successIllustration from '../../assets/notificationsSuccess.svg'
+import errorIllustration from '../../assets/notificationsError.png'
+
+const StatusInfo = React.memo(function StatusInfo({
+  error,
+  title,
+  description,
+  actionText,
+  onAction,
+}) {
+  const theme = useTheme()
+
+  return (
+    <div
+      css={`
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
```

Comment:
Same comment from other reviews, since the div below encapsulates all content and it's already centering all children, this styles are not needed here.

Reply:
I guess the other question is, do we need this parent `div` anymore with the next `div`? It looks like this one's just adding padding.